### PR TITLE
[s8s] Add lib wrapper to use from outside

### PR DESCRIPTION
**Pull request by yoannmoinet**
### What and why?

Using `executeTests` from outside is quite complex and requires a somewhat big bootstrap we should not have to worry about when using this as a library.

### How?

Create a wrapper around `executeTests` called `execute` (open on the naming).
This wrapper will handle the reporters and configuration defaults.

Also tested locally with 
- https://github.com/DataDog/web-ui/pull/77588

### Review checklist

- [x] Feature or bugfix MUST have appropriate tests (unit, integration)

### CODE REVIEW

**Review by etnbrd — APPROVED** [review]
LGTM 👍

**Comment by etnbrd** [comment]
I was about to leave a comment about the naming of `execute` on the account that it's quite broad, and already called like that in the command. But actually, it kind of makes sense to have the same naming between the command and the lib, as they do roughly the same thing (except for some config handling and reporting) on the same level (wrapping `executeTests` and `renderResults`).

File: src/commands/synthetics/run-test.ts [hunk 1]
```diff
@@ -218,3 +230,61 @@ export const getTestsList = async (
...
+    reporters,
+    runId,
+    suites,
+  }: {
+    jUnitReport?: string
+    reporters?: (SupportedReporter | Reporter)[]
+    runId?: string
+    suites?: Suite[]
+  }
+) => {
```
Comment (Drarig29) [inline src/commands/synthetics/run-test.ts:247]: nit: could you add an explicit return type here? (we usually have a lint rule for that but this repo doesn't have one)

**Review by Drarig29 — COMMENTED** [review]
Good idea, I like the `(SupportedReporter | Reporter)[]` part 👍 

Only nits on my side

**Review by Drarig29 — APPROVED** [review]

**Pull request merged**